src/IO/H5/Python/Dat.cpp

```diff
@@ -0,0 +1,71 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <Python.h>
+#include <boost/python.hpp>
+#include <boost/python/stl_iterator.hpp>
+#include <boost/python/suite/indexing/vector_indexing_suite.hpp>
+#include <hdf5.h>
+#include <sstream>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "DataStructures/Matrix.hpp"
+#include "IO/H5/Dat.hpp"
+#include "IO/H5/Header.hpp"
+#include "IO/H5/OpenGroup.hpp"
+#include "IO/H5/Python/Numpy.hpp"
+#include "IO/H5/Python/ToNumpy.hpp"
+#include "IO/H5/Version.hpp"
+
+namespace bp = boost::python;
+
+template <typename T>
+inline std::vector<T> py_list_to_std_vector(const bp::object& iterable) {
```

Comment:
It probably makes sense to factor the `std_vector_to_py_list` type functions into a separate file in `src/PythonBindings` so they can be called from multiple places. The vector one is already used twice just in this PR and I expect that number to increase as we have more wrappers :)